### PR TITLE
Less weird light bulbs.

### PR DESCRIPTION
**Pull request by chaoko99**
## Description of changes

Emissive light bulbs. The dream is to remove the light shaft and replace it with filters. That'd be cool. I don't really know how mutable appearances work though.

## Why and what will this PR improve
Pretty lights with purer colors.

## Authorship
chaoko99 edit

## Changelog

:cl:
tweak: Light bulbs are emissive and color better.
/:cl:

image

### CODE REVIEW

**Comment by MistakeNot4892** [comment]
You committed your .dme and this was previously reverted in the past.

File: html/changelogs/AutoChangeLog-pr-2127.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+author: chaoko99
+delete-after: true
+changes:
+  - imageadd: New combat shield sprite.
```
Comment (MistakeNot4892) [inline html/changelogs/AutoChangeLog-pr-2127.yml]: er

**Review by MistakeNot4892 — CHANGES_REQUESTED** [review]

**Comment by chaoko99** [comment]
will make another attempt when I unfuck my branch stand by

**Comment by chaoko99** [comment]
branch unfucked

**Comment by chaoko99** [comment]
Scope of this PR has widened. Closing until I have something more concrete.

**Pull request closed without merging**